### DISCUSSION TITLE
Exclude actions-user from clones counter increment

### DISCUSSION THREAD

**Discussion by andry81** [community/community#14079] [Actions]
Currently the https://github.com/actions-user user does involved into the clones counter change in git clone command from the pipeline. This distorts the clone statistic and it does mix with the real users clone number. Can it be somehow filtered out?

**Comment by chrispat** [comment]
It may be possible, but I am curious what scenarios this impacts.  I expect any GitHub App that clones a repo would also be included in those statistics.

**Reply by Marychoffel22** [reply]
Nordpass@smtp.com

**Reply by andry81** [reply]
I have a set of GitHub action scripts to collect and accumulate statistic beyond 14days limit. If higher the quantity per day of runs, then higher clones counter increment.

https://github.com/andry81-devops/github-accum-stats

**Reply by chrispat** [reply]
I understand the number is higher. I am trying to understand what you are trying to gather through those statics and how the number of clones including the actions checkouts is hurting your scenario.

**Reply by andry81** [reply]
> I understand the number is higher. I am trying to understand what you are trying to gather through those statics and how the number of clones including the actions checkouts is hurting your scenario.

The point of whole thing is to collect correct statistic in first place. Statistic is not clean, what is can be said here?
Or question is why the statistic needs to be collected? For example, to track a repo progress on user response I think.
